graphql/authorizations.ts

```diff
@@ -401,6 +401,7 @@ export const authorizationModelEnhanceMap: ModelsEnhanceMap = {
             match: nobody,
             _count: nobody,
             waiting_list_enrollment: nobody,
+            lastLogin: nobody,
```

Comment:
I think we could release it to onlyAdminOrOwner. This might also be useful for support